testing/oak_echo_bin/src/main.rs

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![no_std]
+#![no_main]
+#![feature(alloc_error_handler)]
+#![feature(core_c_str)]
+#![feature(core_ffi_c)]
+
+extern crate alloc;
+
+use alloc::boxed::Box;
+#[cfg(all(feature = "raw_server", not(feature = "idl_server")))]
```

Comment:
It might be a bit cleaner to read if you move this and the constant in line 36 into the code block starting at line 49.

Reply:
Done